plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -25,6 +26,9 @@ object ScalafmtPlugin extends AutoPlugin {
   object autoImport {
     val scalafmt = taskKey[Unit]("Format Scala sources with scalafmt.")
 
+    val ScalafmtTag = Tag("scalafmt")
```

Comment:
Just name clarifying

```suggestion
    val ScalafmtParallelExecutionTag = Tag("scalafmt")
```

Reply:
Happy to do it, but I have usability concerns. 

In practice this means that to use this, one would do something like 

```scala
Global / concurrentRestrictions ++= Tags.limit(ScalafmtParallelExecutionTag, 3)
```

AIUI there are three competing concerns:

1. * usability (as above), would pull the name towards `Scalafmt`
2. * maintainability, as "tag" is a bout nondescriptive as it goes, pulls towards `ScalafmtParallelExecutionTag` indeed (or, I'd argue, any solution that makes it obvious within sbt-scalafmt what we're talking about, and you are correct my original code doesn't make it easy) 
3. * discoverability (if it's hidden it can't be discovered)

I have two ideas that possibly improve on your suggestion:

(a) just put it elsewhere, in a way that makes it obvious what it is 
```scala
package org.scalafmt.sbt.ConcurrentRestrictions
import sbt.ConcurrentRestrictions.Tag

object Tags {
  val Scalafmt = Tag("scalafmt")
}
```

(b) rename the word `Tag`in the context of sbt-scalafmt so that a reader of the code knows what's it about *but* keep this closer to what SBT call it in the exposed API/ABI

```diff
@@@above
-import sbt.ConcurrentRestrictions.Tag
+import sbt.ConcurrentRestrictions.{Tag => SbtConcurrentRestrictionsTag}

@@@at line 28

-val ScalafmtTag = Tag("scalafmt")
+val ScalafmtTag = SbtConcurrentRestrictionTag("scalafmt")
```

regardless of (a) or (b) (or your suggestion), I'll rename "ScalafmtTagPack" into "ConcurrentRestrictionsTags" to reduce reader's surprise

WDYT?

Reply:
With suggestion (a) tags usage will look like 

```scala
import org.scalafmt.sbt.ConcurrentRestrictions

...

Global / concurrentRestrictions ++= Tags.limit(ConcurrentRestrictions.Tags.Scalafmt, 3)
```
?

If yes, I am ok with it

Reply:
With suggestion (a), yes.

Suggestion (b) would let user do instead 

```
// no import (autoImport in play)
....
Global / concurrentRestrictions ++= Tags.limit(ScalafmtTag, 3)
```

while we'd keep a very explicit (type/constructor) name within scalafmt

Will go with whichever you pick

Reply:
I think that (a) is better because it provides tag _purpose_ on the user side

Reply:
OK

Reply:
just pushed a slight variation on (a) to avoid having a PascalCase namespace

looking from the user side, I'm still a bit nervous about what happens if another popular package follows the same convention — one will have to do renaming imports then to avoid a collision on the name `ConcurrentRestrictionsTag`.

Reply:
Thank you. I will test and merge it today